website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -2441,5 +2466,8 @@ var Range = ace.require('ace/range').Range;
         chunk.skipLines(2, 1, true);
     }
 
-
 })();
+$(function(){
+  $("wmd-help-button").attr("data-toggle","modal");

```

Comment:
This doesn't appear to be selecting anything, since you're not selecting for a class (with `.`) an ID (with `$`). Can this code be removed?